lib/api.ts

```diff
@@ -51,6 +52,14 @@ export class APIClient {
     return this.request.post(`${this.apiRoot}/send/email`, req.message);
   }
 
+  getCustomersByEmail(email: string) {
+    if (typeof email !== 'string' || !email) {
```

Comment:
We prefer the [`isEmpty`](https://github.com/customerio/customerio-node/blob/c064c61dc2131f37a7d08c33ac0588ac9884295e/lib/utils.ts#L3) util to the `!` operator to catch cases where the value is an empty string

Reply:
done!